.gitignore

```diff
@@ -6,7 +6,7 @@ results/
 *model*
 *.pt
 *pth
-
+*.ds_store

```

Comment:
as this is a macOS related file, this is fine, but some OS (like linux) are case sensitive, so just keep that it mind when adding stuff to .gitignore